### PR TITLE
add NVIDIA to institutional partners

### PR DESCRIPTION
**Pull request by kkraus14**
Related to https://github.com/conda-forge/conda-forge.github.io/pull/1200

I believe NVIDIA has also provided monetary funding to conda-forge. Unsure if that should be captured here or not, but given I'm an inherently biased source 😅 would defer to others thoughts.

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks Keith! I think we want to add RAPIDS as well.

**Comment by beckermr** [comment]
Yes. We should capture that info @kkraus14. I am happy to add a fiscal sponor section if that works. 

Please do add anyone else @jakirkham!

**Comment by kkraus14** [comment]
> Thanks Keith! I think we want to add RAPIDS as well.

RAPIDS isn't an institution, it's another open source community where I don't think it's appropriate to include it here.

**Comment by jakirkham** [comment]
That makes sense. I don't have strong feelings here. Am basing this on @datametrician's suggestion to me previously. Maybe an alt logo would make more sense ( https://github.com/conda-forge/conda-forge.github.io/pull/1028#issuecomment-615414550 )?

File: src/inst_partners.yaml [hunk 1]
```diff
@@ -66,6 +66,18 @@ QuantStack:
...
+   Description: NVIDIA provides developer time
+   Developers:
+     - jakirkham
+     - kkraus14
+   Logo URL:
+     - https://s3.amazonaws.com/cms.ipressroom.com/219/files/20149/544a0d86f6091d6699000060_NVLogo_2D/NVLogo_2D_362acb00-8e1b-476b-9662-9fe138a4a920-prv.jpg
+   Site URL:
+     - https://www.nvidia.com
+   contribution type:
+     - dev support
```
Comment (beckermr) [inline src/inst_partners.yaml:79]:
```suggestion
     - dev support
     - fiscal support
```

**Comment by kkraus14** [comment]
@beckermr this should be good to go now pending that the updated URL still renders as expected. Feel free to merge when you're ready.

**Pull request merged**